stdlib/c/compile.mc

```diff
@@ -148,7 +163,7 @@ lang MExprCCompile = MExprAst + CAst
   -- Intentionally left blank
 
   -- Entry point
-  sem compile (typeEnv: [(Name,Type)]) =
+  sem compile (typeEnv: [(Name,Type)]) (compileOptions : CompileCOptions) =
```

Comment:
The `(compileOptions : CompileCOptions)` addition is basically the only interface change, right?

Reply:
Yes.

Reply:
Or no, the `compileCEnvEmpty` function now also expects a `CompileCOptions` record as a parameter.